### PR TITLE
💰 Pricing updates via Tina

### PR DESCRIPTION
**Pull request by isaaclombardssw**
Prospective change to the pricing page.

<details>
### General Contributing:
- [ ] Have you followed the guidelines in our [Contributing document](https://github.com/tinacms/tina.io/blob/master/CONTRIBUTING.md)?
- [ ] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?

### All New Content Submissions: (To be confirmed by reviewer)

- [ ] Title is short & specific
- [ ] Headers are logically ordered & consistent
- [ ] Purpose of document is explained in the first paragraph
- [ ] Procedures are tested and work
- [ ] Any technical concepts are explained or linked to
- [ ] Document follows structure from templates
- [ ] All links work
- [ ] The spelling and grammar checker has been run
- [ ] Graphics and images are clear and useful
- [ ] Any prerequisites and next steps are defined.
</details>

### CODE REVIEW

File: content/blocksPages/pricing.json [hunk 1]
```diff
@@ -52,20 +57,15 @@
...
         },
         {
-          "name": "Team",
-          "description": "For growing teams\n",
-          "price": "$29",
-          "interval": "/ month",
+          "name": "Team Plus",
+          "description": "For bigger teams\n",
+          "price": "$49",
+          "interval": " forever",
```
Comment (joshbermanssw) [inline content/blocksPages/pricing.json:63]: This should be monthly?

Comment (isaaclombardssw) [reply content/blocksPages/pricing.json]: Fixed.

**Review by joshbermanssw — APPROVED** [review]
lgtm

File: content/blocksPages/pricing.json [hunk 2]
```diff
@@ -90,38 +103,38 @@
...
-              "icon": "AiOutlineUsergroupAdd",
               "description": ""
             },
             {
               "name": "3 Roles (Beta)",
               "icon": "BiBadge"
             },
             {
-              "name": "Premium Support",
+              "name": "Priority Support",
```
Comment (wicksipedia) [inline content/blocksPages/pricing.json:119]: should be Premium Business Support

File: content/blocksPages/pricing.json [hunk 3]
```diff
@@ -74,11 +74,24 @@
             {
               "name": "Forum Support",
```
Comment (wicksipedia) [inline content/blocksPages/pricing.json:75]: should be Team Support

**Pull request merged**